ui/app/src/components/poolList/SinglePool.vue

```diff
@@ -153,7 +151,13 @@ export default defineComponent({
           </div>
         </div>
       </div>
-
+      <div class="section">
+        <div class="info">
+          <h3 class="mb-2">Liquidity provider rewards</h3>
+          <p class="text--small mb-2">Liquidity providers earn a 0.3% fee on all trades proportional to their share of the pool. Fees are added to the pool, accrue in real time and can be claimed by withdrawing your liquidity.</p>
```

Comment:
[Per Casey](https://sifchain.slack.com/archives/G01BX71QY1F/p1610477848293900?thread_ts=1610471699.290600&cid=G01BX71QY1F), change copy and link in `<p>` below to: 
"Liquidity providers earn a percentage fee on all trades proportional to their share of the pool. Fees are added to the pool, accrue in real time and can be claimed by withdrawing your liquidity. To learn more, reference of documentation <a href="https://app.gitbook.com/@sifchain/s/sifchain/roles/liquidity-providers">here</a>."